indico/modules/categories/blueprint.py

```diff
@@ -26,6 +27,10 @@
 
 # Management
 _bp.add_url_rule('/manage/', 'manage_content', RHManageCategoryContent)
+_bp.add_url_rule('/manage/settings/', 'manage_settings', RHManageCategorySettings, methods=('POST', 'GET'))
+_bp.add_url_rule('/manage/settings/upload_icon', 'upload_icon', RHSettingsIconUpload, methods=('POST',))
+_bp.add_url_rule('/manage/settings/delete_icon', 'delete_icon', RHSettingsIconDelete, methods=('DELETE',))
 
 # Display
+_bp.add_url_rule('/icon', 'display_icon', RHSettingsIconDisplay)
```

Comment:
it's nice for cache busting, and doesn't cost anything (we have it for event logos/stylesheets too) - if you don't want it i'd at least remove it from `icon_url` too so it doesn't go into the query string